libcloudforensics/providers/kubernetes/base.py

```diff
@@ -186,6 +188,34 @@ def ListPods(self, namespace: Optional[str] = None) -> List['K8sPod']:
         for pod in pods.items
     ]
 
+  def ExternalIP(self) -> str:
+    """Returns the external IP of this node.
+
+    Returns:
+      str: The external IP of this node.
```

Comment:
what if a node (type gce instance) has more than one external IP? do you want to get only the first one or should this returns a list rather than single string?

Reply:
Changed to list return type